test/avio.jl

```diff
@@ -15,31 +13,25 @@ swapext(f, new_ext) = "$(splitext(f)[1])$new_ext"
 
 println(STDERR, "Testing file reading...")
 
-if !isdefined(Main, :UFixed8)
-    UFixed8 = Ufixed8
-end
-
 function notblank(img)
-    all(Images.green(img) .== 0x00uf8) || all(Images.blue(img) .== 0x00uf8) || all(Images.red(img) .== 0x00uf8) || maximum(reinterpret(UFixed8, img)) < 0xcfuf8
-end
-
-if isdefined(Images, :load)
-    imload = Images.load
-else
-    imload = Images.imread
+    all(c->green(c) == 0, img) || all(c->blue(c) == 0, img) || all(c->red(c) == 0, img) || maximum(rawview(channelview(img))) < 0xcf
 end
 
 for name in VideoIO.TestVideos.names()
     is_apple() && startswith(name, "crescent") && continue
     println(STDERR, "   Testing $name...")
 
     first_frame_file = joinpath(testdir, swapext(name, ".png"))
-    first_frame = imload(first_frame_file) # comment line when creating png files
+    first_frame = load(first_frame_file) # comment line when creating png files
 
     f = VideoIO.testvideo(name)
     v = VideoIO.openvideo(f)
 
-    img = read(v, Image)
+    if size(first_frame, 1) > v.height
+        first_frame = first_frame[1+size(first_frame,1)-v.height:end,:]
+    end
```

Comment:
I'm curious what necessitated this change?  Tests pass for me without it (using `ImageMagick` on a Mac), and the images being tested against were pulled directly from the videos that are being read, so they should be exactly the same size.